README.md

```diff
@@ -32,12 +32,20 @@ The project uses the latest versions of the tools and libraries.
  
  ### Components included in the app
   - Auth module: define authentication
-  - Locale selector for SPA: support multiple languages
-  - Theme: integrate with a theme
-  - Entitlements: configure entitlements for different scenarios
+  - Locale selector for SPA: support multiple languages. Check the example codes [here](https://github.com/Backbase/golden-sample-app/tree/master/apps/golden-sample-app/src/app/locale-selector)
+  - Entitlements: configure [entitlements](https://community.backbase.com/documentation/foundation_angular/latest/entitlements) for different scenarios. Check the example code in the [`app.component.html`](https://github.com/Backbase/golden-sample-app/blob/738df3de9fb15f96c9fa2017d00269ced63bebaa/apps/golden-sample-app/src/app/app.component.html#L50) and [`entitlementsTriplets.ts`](https://github.com/Backbase/golden-sample-app/blob/master/apps/golden-sample-app/src/app/services/entitlementsTriplets.ts)
   - Configure journeys
+    - This is [an example](https://github.com/Backbase/golden-sample-app/blob/master/libs/transactions/src/lib/services/transactions-journey-config.service.ts) is how the journey configuration might be defined in the application 
```

Comment:
There is an extra `is` that can be replaced by 'of'.  "This is an example **is** how" => "This is an example of how"